src/styles/_catalog.scss

```diff
@@ -1,6 +1,8 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
there should be 4 colums starting at 1024, your 4 columns start at 944:
<img width="1061" alt="image" src="https://user-images.githubusercontent.com/37161315/157227173-48573731-56ee-4259-b472-776283e808e8.png">
same with 3 columns that should start at 768:
<img width="724" alt="image" src="https://user-images.githubusercontent.com/37161315/157227042-2e66d67c-e80e-49cf-9cff-07d95f543b5c.png">
same with 2 columns:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/37161315/157227135-70014ee1-4b85-4193-9251-98f887140e17.png">